data/locale/jack_mixer-fr.po

```diff
@@ -0,0 +1,677 @@
+# #-#-#-#-#  jack_mixer-fr.po (jack_mixer 15.1)  #-#-#-#-#
```

Comment:
Can you remove this line, please?

Reply:
I can do this, sure.
Note that I've been putting it because it's present in the jack_mixer-de.po file you did yourself.
Should that be removed there as well then?

Reply:
Yes, that's an artefact of msgmerge. I'll remove it in the German `.po` file.

Reply:
OK, removed from the fr.po